src/app/organization/list/organization-list.component.scss

```diff
@@ -0,0 +1,295 @@
+            }
+            .card-metainfo {
+              font-size: 0.9em;
+              float: right;
+              color: $main-color;
+
+              .card-location {
+                margin-left: 1em;
+                margin-right: 1em;
+              }
+            }
+          }
+
+          .card-content {
+            padding:0;
+            width: 100%;
+            display:table;
+            .card-avatar {
+              padding: 5px;
+              width: 40%;
+              display:table-cell;
+              vertical-align: top;
+              min-height: 250px;
+              > img {
+                width: 100%;
+                vertical-align: middle;
+              }
+            }
+            .card-info {
+              width: 60%;
+              display:table-cell;
+              vertical-align: top;
+              position: relative;
+              .description {
+                text-align: justify;
+                font-family: Raleway;
+                padding: 11px;
+                font-size: 10px;
+              }
+              h5 {
+                font-size: 1em;
+                font-weight: bold;
+                text-align: center;
+                color: $main-color;
+              }
+
+              .skills-list-wrapper {
+                padding: 5px 25px 5px 25px;
+                ul {
+                  li.chip {
+                    color: $main-color;
+                    border: solid $line-sep-width $header-bg-color;
+                    background-color: #fff;
+                    border-radius: 0px;
+                    margin-left: 0.25em;
+                    margin-right: 0.25em;
+                    min-width: 27%;
+                    text-align: center;
+
+                    &:hover {
+                      background-color: $bg-line-color;
+                    }
+                  }
+
+                }
+              }
+              .card-info-bottom {
+                background-color: $sec-bg-color;
+                padding-left: 25px;
+                vertical-align: bottom;
+                position: absolute;
+                left: 0;
+                right: 0;
+                bottom: 0;
+
+                .org-name {
+                  line-height: 50px;
+                  font-size: 1em;
+                  font-weight: bold;
+                  text-align:center;
+                  color: $main-color;
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .pagination {
+    border-top: solid $line-sep-width $line-sep-color;
+    border-bottom: solid $line-sep-width $line-sep-color;
+    .pages-list-wrapper {
+      margin: 0;
+      padding: 0;
+      
+    }
+  }
+}
+
+pagination-controls /deep/ .ng2-pagination  {
+  font-size: 0px;
+  margin: 0;
+  padding: 0 ;
+  margin-top: - $line-sep-width;
+  margin-bottom: - 2 * $line-sep-width;
+  text-align: center;
+  line-height: 50px;
+}
+pagination-controls  /deep/ .ng2-pagination li {
```

Comment:
these `/deep/` styles should be moved to the global `styles.s?css` file, they do not require `/deep/`